src/pages/edit-account/edit-account.html

```diff
@@ -0,0 +1,39 @@
+<!--
+  Generated template for the EditAccount page.
```

Comment:
Should remove auto-generated comments since they will affect our quality metrics